### PR TITLE
Feat V0.35: Improvements to ModelArchiveModelProvider, and addition of more ModelArchive data sets

### PR DESCRIPTION
**Pull request by piehld**

### CODE REVIEW

File: rcsb/utils/insilico3d/ModelArchiveModelProvider.py [hunk 1]
```diff
@@ -92,17 +94,29 @@ def __reload(self, **kwargs):
...
+                        # "bulkFileName": "",
+                        # "numModelsTotal": ,
+                    },
+                    "ma-asfv-asfvg": {
+                        # "bulkFileName": "ma-asfv-asfvg.zip",
+                        # "numModelsTotal": ,
+                    },
+                    "ma-t3vr3": {
+                        # "bulkFileName": "ma-t3vr3.zip",
+                        # "numModelsTotal": ,
```
Comment (brindakv) [inline rcsb/utils/insilico3d/ModelArchiveModelProvider.py:119]: "numModelsTotal" seems to have been replaced everywhere else. Is it intentional to leave it here commented?

File: rcsb/utils/insilico3d/ModelArchiveModelProvider.py [hunk 2]
```diff
@@ -144,8 +158,7 @@ def __reload(self, **kwargs):
                 for dataSet, pathD in modelArchiveRequestedDatasetD.items():
                     try:
                         sD = {}
-                        dataSetNumModels = pathD.get("numModelsTotal")
-                        numModelsToDownload = pathD.get("numModels", dataSetNumModels)  # Used for testing purposes, defaults to total number of models
+                        numModelsToDownload = pathD.get("numModels", None)  # Used for testing purposes, defaults to total number of models
                         bulkFileName = pathD.get("bulkFileName", None)
```
Comment (brindakv) [inline rcsb/utils/insilico3d/ModelArchiveModelProvider.py:162]: Am I right in understanding that neither full download or test download use "bulkFileName".

**Review by brindakv — COMMENTED** [review]
Thanks, @piehld. Please see my comments below.

File: rcsb/utils/insilico3d/ModelArchiveModelProvider.py (hunk 2 above)
Comment (piehld) [reply rcsb/utils/insilico3d/ModelArchiveModelProvider.py:162]: Currently, no. I've left the functionality in there to perform the bulk file download once bulk files are available for everything, but for now I'm defaulting things to perform the individual-file download instead.

File: rcsb/utils/insilico3d/ModelArchiveModelProvider.py (hunk 1 above)
Comment (piehld) [reply rcsb/utils/insilico3d/ModelArchiveModelProvider.py:119]: Good point, I'll clean that up too. Thanks.

**Pull request merged**